doc/Language/grammar_tutorial.pod6

```diff
@@ -621,16 +559,14 @@ And this is the grammar and grammar actions that got us there, to recap:
     }
     =end code
 
```

Comment:
We should also include an alternate version that splits the `data` into smaller pieces by just using the grammar methods. The whole time that was in my mind like "why do we need an action for this"? So perhaps we can quell questions by just including the alternate as a side note.